src/style.css

```diff
@@ -74,7 +74,7 @@
 }
 
 /* Reset buttons */
-.rdp-root button {
+.rdp-root :where(.rdp-day_button, .rdp-button_previous, .rdp-button_next) {
```

Comment:
```suggestion
.rdp-button-reset
```

Reply:
Yes, I thought so too, but then we need to define two classes for the `Button` component, right? But every component gets its classes from `UI` enum. How do you plan on adding this class? Do you mean like this?

```jsx
<components.Button
    type="button"
    className={[classNames[UI.ButtonPrevious], 'rdp-button-reset'].join(' ')}
    tabIndex={previousMonth ? undefined : -1}
    disabled={previousMonth ? undefined : true}
    aria-label={labelPrevious(previousMonth, labelOptions)}
    onClick={handlePreviousClick}
  >
```

Reply:
@hsnaydd yes, it should come with a relative entry in the `UI` enum:

```tsx
className={[classNames[UI.ButtonPrevious], classNames[ButtonReset]].join(' ').trim()}
```

not so quick as it seems (there are some files to change), but if you feel - give a try!

Reply:
I don't think this is the right solution. if I want to change the `ButtonPrevious` class via `classNames` props `ButtonReset` will still be applied. If I want to set padding to the `Button` with my new class I still need to increase the specificity because `ButtonReset` may overwrite. Yes, I can set the `button_reset` to `undefined` to prevent this but it is not predictable.

Reply:
I did, but, I don't understand why you want to avoid the `:where` pseudo selector. What will change for Tailwind users? If you care about the specificity, it is at the same level as what we are writing now. Also if you don't scope with `.rdp-root` the specificity value will be zero.

Reply:
> I did, but, I don't understand why you want to avoid the :where pseudo selector. What will change for Tailwind users? 

It's just about keeping it simple. I've seen complex selectors may cause confusions to some users (see recent support requests about tailwind).